cluster/packages.sls

```diff
@@ -40,18 +39,21 @@ install_cluster_packages:
 
 {% endif %}
 
-{% if grains['cloud_provider'] == 'microsoft-azure' %}
+# The next 2 onlyif statements can be improved when salt version 3000 is used
+# https://docs.saltstack.com/en/latest/ref/states/requisites.html#onlyif
 install_additional_packages_azure:
   pkg.installed:
+    - onlyif: cat /etc/salt/grains | grep "cloud_provider:.*microsoft-azure"
```

Comment:
I guess this statement can be passed as argument under `onlyif` but this is same

Reply:
Yes, it might be refined a bit. I think it's fine by now